enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -11,18 +11,21 @@
 from enterprise_access.apps.subsidy_access_policy.mocks import catalog_client, group_client, subsidy_client
 
 
+class PolicyManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(proxy_class_name=self.model.__name__)
+
+
 class SubsidyAccessPolicy(TimeStampedModel):
     """
     Tie together information used to control access to a subsidy.
     This abstract model joins group, catalog, and access method.
 
     .. no_pii:
     """
-    class Meta:
-        """
-        Metaclass for SubsidyAccessPolicy.
-        """
-        abstract = True
+
+    PROXY_FIELD_NAME = 'proxy_class_name'
+    proxy_class_name = models.CharField(max_length=64)
```

Comment:
the name is too django centric ... can we call this policy_name or something similar?